ext/File-Find/t/find.t

```diff
@@ -19,8 +18,7 @@ BEGIN {
         require File::Spec::Unix;
         @File::Spec::ISA = 'File::Spec::Unix';
     }
-    require File::Find;
-    import File::Find;
+    use File::Find;
```

Comment:
this is incorrect and not going to honor the `@INC` setup above in the same block